style.css

```diff
@@ -0,0 +1,154 @@
+* {
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+}
+
+a {
+  cursor: pointer;
+}
+
+.hide {
+  display: none;
+}
+
+.social-media {
+  margin: 5% 1%;
+  list-style: none;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.social-media > li > a {
+  margin: 0 5px;
+}
+
+#toolbar {
+  background: #3c3a39;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 15px 6%;
+}
+
+#logo {
+  color: #fff;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 1.2rem;
+  text-decoration: none;
+}
+
+#close-navlist {
+  position: absolute;
+  top: 15px;
+  right: 20px;
+}
+
+#headline {
+  background: #1c1a19 url('ASSETS /illustration-header-mobile.svg') no-repeat right bottom;
```

Comment:
- I think the problem with the background image lies here as you don't have a folder called "ASSETS" I think you meant files 👍